src/executor/plan_executor.cpp

```diff
@@ -64,19 +63,19 @@ static void CompileAndExecutePlan(
     codegen::QueryCache::Instance().Add(plan, std::move(compiled_query));
   }
 
-  auto on_query_result =
-      [&on_complete, &consumer](executor::ExecutionResult result) {
-        std::vector<ResultValue> values;
-        for (const auto &tuple : consumer.GetOutputTuples()) {
-          for (uint32_t i = 0; i < tuple.tuple_.size(); i++) {
-            auto column_val = tuple.GetValue(i);
-            auto str = column_val.IsNull() ? "" : column_val.ToString();
-            LOG_TRACE("column content: [%s]", str.c_str());
-            values.push_back(std::move(str));
-          }
-        }
-        on_complete(result, std::move(values));
-      };
+  auto on_query_result = [&on_complete,
+                          &consumer](executor::ExecutionResult result) {
+    std::vector<ResultValue> values;
```

Comment:
There isn't any logic change here, but are you using `clang-format-3.6` for formatting?